### DISCUSSION TITLE
Open recently closed tab shortcut

### DISCUSSION THREAD

**Discussion by sglbl** [sumatrapdfreader/sumatrapdf#3387] [Ideas]
I think _open recently closed tab_  by pressing **Ctrl Shift T** would be a nice shortcut like in the browsers.

**Comment by tactilis** [comment]
Ctrl+Shift+T does that already.

You may need the pre-release version:

![image](https://user-images.githubusercontent.com/4060092/227773579-13b91336-2795-4422-8e52-ce5a318c868a.png)

**Comment by kjk** [comment]
Yes, you need pre-releaser to get this feature and more: https://github.com/sumatrapdfreader/sumatrapdf/discussions/2735
